addons/point_of_sale/static/src/app/screens/payment_screen/payment_screen.js

```diff
@@ -279,9 +279,9 @@ export class PaymentScreen extends Component {
 
             // 2. Invoice.
             if (this.shouldDownloadInvoice() && this.currentOrder.is_to_invoice()) {
-                if (this.currentOrder.account_move) {
+                if (this.currentOrder.raw.account_move) {
```

Comment:
do we need to use the raw ? I think we just fix the issue ?

Reply:
the problem is that we won't load the account.move model when the point_of_sale is only installed, and then the account_move would be undefined, but we have the id in the raw.